test-framework/test-suites/integration/tests/report/test_report_host.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+
+
+@pytest.mark.usefixtures("revert_database")
+class TestReportHost:
+    def test_report_host_with_default_network_behaviour():
+        result = host.run(f'stack add host backend-0-0')
+        assert result.rc == 0
+        result = host.run(f'stack add host interface backend-0-0 interface=eth0 ip=10.10.11.11 network=private')
+        assert result.rc == 0
+        result = host.run(f'stack add host backend-0-1')
+        assert result.rc == 0
+        result = host.run(f'stack add host interface backend-0-1 interface=eth1 ip=10.10.11.15 network=private default=True')
+        assert result.rc == 0
+        result = host.run(f'stack report host')
+        assert result.rc == 0
+        print("-----")
+        print(result.stdout.strip())
+        print("-----")
+
+    def test_report_host_with_shortname_behaviour():
+        result = host.run(f'stack add host backend-0-0')
+        assert result.rc == 0
+        result = host.run(f'stack add host interface backend-0-0 interface=eth0 ip=10.10.11.11 network=private')
+        assert result.rc == 0
+        result = host.run(f'stack add host backend-0-2')
+        assert result.rc == 0
+        result = host.run(f'stack add host interface backend-0-2 interface=eth2 ip=10.10.11.15 network=private')
+        assert result.rc == 0
+        result = host.run(f'stack set host interface options backend-0-2 interface=eth2 options=shortname')
+        assert result.rc == 0
+        result = host.run(f'stack report host')
+        assert result.rc == 0
+        print("-----")
+        print(result.stdout.strip())
+        print("-----")
+
+    def test_report_host_with_network_zone_behaviour():
+        result = host.run(f'stack add host backend-0-0')
```

Comment:
What's the purpose of adding backend-0-0 in this test?  It doesn't appear to be what's being tested?  If it is, add a comment.